search_similar_entities/facetedSearchEndpoint.rb

```diff
@@ -0,0 +1,161 @@
+#QuerySimilar Endpoint
+
+#Builds complex query for"Query Similar" functionality; returns a link to the search page, with a preseeded query looking for all entities of type testEntity that share certain attributes with the current "testEntity" entity, or alternatively, the same query but with preselected facets
+
+# Verify params are OK
+raise "Missing required parameter: id" unless params[:id]
```

Comment:
yeah, you're right.  Sorry - I was thinking of another web framework at the time!